oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBIndexInstaller.java

```diff
@@ -60,9 +61,9 @@ public boolean isExists(Model model) throws StorageException {
             // then check entity schema
             if (metadata.findRemoteSchema(c).isPresent()) {
                 // register models only locally but not remotely
-                if (model.isTimeSeries() && model.isRecord()) { // stream
+                if (model.isRecord()) { // stream
                     MetadataRegistry.INSTANCE.registerStreamModel(model, config, configService);
-                } else if (model.isTimeSeries() && !model.isRecord()) { // measure
```

Comment:
I don't think this could be removed. Not `model.isTimeSeries()` model exists.

Reply:
I don't think we should register UI_TEMPLATE as a measure, right?

Reply:
We've already checked at the beginning of `isExists`,

```java
@Override
    public boolean isExists(Model model) throws StorageException {
        if (!model.isTimeSeries()) {
            return true;
        }
        ...
}
```

So I think we can safely ignore other conditions?